packages/protocol/lib/web3-utils.ts

```diff
@@ -387,6 +393,28 @@ export function getFunctionSelectorsForContract(contract: any, contractName: str
   return selectors
 }
 
+export function checkInheritance(baseContractName: string, derivativeContractArtifact: any) {
+  return derivativeContractArtifact.ast.nodes.find(
+    (astNode: any) => astNode.nodeType === 'ImportDirective' && (astNode.file as string).endsWith(`${baseContractName}.sol`)
```

Comment:
Ditto with node.

Reply:
The node can't be typed because we also use truffle artifacts